### PR TITLE
docs(samples): cleanup quickstart and metrics samples

### PR DESCRIPTION
**Pull request by syedashrafulla**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [N/A] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-monitoring/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [X] Ensure the tests and linter pass
- [X] Code coverage does not decrease (if any source code was changed)
- [X] Appropriate docs were updated (if necessary)

Fixes quickstart and metrics samples to run with any optional parameters truly optional and using best practices. 🦕

### CODE REVIEW

File: samples/metrics.js [hunk 1]
```diff
@@ -30,10 +30,7 @@ async function createMetricDescriptor(projectId) {
   // Creates a client
   const client = new monitoring.MetricServiceClient();
 
-  /**
```
Comment (JustinBeckwith) [inline samples/metrics.js:33]: Greetings!  These are here on purpose :) We don't want to assume developers running the same have the `GOOGLE_CLOUD_PROJECT` environment variable set on their current process.  We cover a lot of the motivation for this here:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head

**Review by JustinBeckwith — CHANGES_REQUESTED** [review]

File: samples/metrics.js (hunk 1 above)
Comment (syedashrafulla) [reply samples/metrics.js:33]: What confuses me is that this is inside a function where projectId is optional, so when I uncomment that line I get

`SyntaxError: Identifier 'projectId' has already been declared`

How should I fix this line while still avoiding the assumption of the presence of environment variables?

Comment (JustinBeckwith) [reply samples/metrics.js:33]: For the code you have now, that makes sense.  You want to leave those sections exactly as they were :) The projectId is passed into the `main` function from the outside.  You should be able to return all the projectId code that was modified back to it's original form.

Comment (syedashrafulla) [reply samples/metrics.js]: I've reverted the `projectId` code back to its original form. From this state, I then uncomment this line so it goes from
> // const projectId = 'YOUR_PROJECT_ID';
to
> const projectId = 'YOUR_PROJECT_ID';

Then I run
> node samples/metrics.js create
and I get
> SyntaxError: Identifier 'projectId' has already been declared

**Review by JustinBeckwith — APPROVED** [review]

**Pull request closed without merging**